src/CmsAdmin/CategoryController.php

```diff
@@ -281,7 +303,9 @@ public function sortAction(Request $request)
         //sortowanie
         foreach ($order as $order => $id) {
             //brak rekordu o danym ID
-            if (null === ($record = (new CmsCategoryQuery())->findPk($id))) {
+            if (null === ($record = (new CmsCategoryQuery())
```

Comment:
co zyskujemy zapisując to tak:

`if (null === ($record = (new CmsCategoryQuery())`
`->whereTemplate()->like($this->scopeConfig->getName() . '%')`
`    ->findPk($id))) {`
`        continue;`
`    }`

a nie tak:

`$record = (new CmsCategoryQuery())`
`    ->whereTemplate()->like($this->scopeConfig->getName() . '%')`
`    ->findPk($id);`
`if (null === $record) {`
`        continue;`
`}`

?

niestety, wycina whitespace więc nie widać tak dobrze różnicy ;)
i tu i tu jest 5 linijek, tylko w drugim jest mniej nawiasów i od razu widać co kiedy się dzieje